### ISSUE TITLE
Localization

### ISSUE THREAD

**Issue by StefanBrand** [WeAreFairphone/landing-page#1]
How to deal with different languages?

**Comment by duke1102** [comment]
Hey, not entirely sure about the complete features of Jekyll yet, but I'd assume you could probably create localized versions of markdown files and then maybe just implement a selection box on the website to easily change around the language.

**Comment by Ryuno-Ki** [comment]
Hi, thanks for the question.

Although I am not on the forum, I read it.

I would follow the same approach [mozilla/activate.mozilla.community](https://github.com/mozilla/activate.mozilla.community) is going, i.e. would pull the translation in from a [data file](https://jekyllrb.com/docs/datafiles/) or [include](https://jekyllrb.com/docs/includes/).
(Disclaimer: I am a Mozillian and contributed to that project.)

Do you have a preferation about how the URL should look like?

**Comment by Ryuno-Ki** [comment]
Speaking of the forum: https://forum.fairphone.com/t/online-portal-for-fairphone-community/28956/11
Which posts would there be? Because there is a [ruby lib for talking to the Discourse API](https://github.com/discourse/discourse_api) (which is one reason why I went for Jekyll instead of, say, Hugo).

**Comment by StefanBrand** [comment]
> Do you have a preferation about how the URL should look like?

In general I was thinking about `fairphone.community/angels` (for example). Is it possible that we have `fairphone.community/en/angels` and that the language is automatically inserted?

**Comment by Ryuno-Ki** [comment]
Done.

I'll have to add the language switcher later on (I'll update todo part of README next).
Note, that `en` is the default language at the moment.
I had to tweak [jekyll-seo-tag](/jekyll/jekyll-seo-tag) to work with [jekyll-multiple-languages-plugin](/Anthony-Gaudino/jekyll-multiple-languages-plugin) a bit. That is, our default theme ([minima](/jekyll/minima)) has `jekyll-seo-tag` already included, so I had to disable the `<title>` generation to make it translateable.

Browse http://localost:4000/de/angels/ for example.
